pkg/asset/manifests/openshift.go

```diff
@@ -258,6 +259,38 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 		assetData["99_private-cluster-outbound-service.yaml"] = applyTemplateData(privateClusterOutbound.Files()[0].Data, templateData)
 	}
 
+	if installConfig.Config.Azure.IsARO() {
```

Comment:
I think we need to check `installConfig.Config.Azure` for nil.

Reply:
Good catch! Updated.